README.md

```diff
@@ -1,23 +1,20 @@
 # ArcGIS Earth Automation API
-ArcGIS Earth Automation API provides a standard interface for 3rd party applications that support REST to automate and communicate with ArcGIS Earth. The Automation API is based on [ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.1). It provides a fast and reliable way for communications with ArcGIS Earth.
+ArcGIS Earth Automation API provides a standard interface for third-party applications that support REST to automate and communicate with ArcGIS Earth. The Automation API is based on [ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.1). It provides a fast and reliable way for communications with ArcGIS Earth.
```

Comment:
Change the ASP.NET Core link to https://learn.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host